### ISSUE TITLE
Getting a 404 when trying to publish package from monorepo

### ISSUE THREAD

**Issue by mathix420** [morlay/bunpublish#4]
Package name: `@plsr/nuxt-layer-base`
- `plsr` is an org 
- I used this command `bunx @morlay/bunpublish` in the root of the monorepo
- [Source code](https://github.com/pulsar-inc/nuxt-layers) (it's a bit of a mess currently 😅)

![image](https://github.com/morlay/bunpublish/assets/37625778/e414dc79-2e60-4063-b3b9-440deac570fb)

**Comment by migueldamota** [comment]
I am getting the same error for my monorepo +1

**Comment by migueldamota** [comment]
I managed to solve this. I updated my `bunfig.toml` to not use

```toml
[install.scopes]
scope = { ... }
```

instead use

```toml
[install]
registry = { url = "https://registry.npmjs.org", token = "$NPM_TOKEN" }
```

This works now perfectly fine for me 👍

**Comment by pipinet** [comment]
@morlay  i got same issue.

```bash
@mesh/lang:publish: 0.0.0
90 |         // www-authenticate header.
91 |         throw new errors.HttpErrorAuthOTP(
92 |           method, res, parsed, opts.spec
93 |         )
94 |       } else {
95 |         throw new errors.HttpErrorGeneral(
                   ^
HttpErrorGeneral: 404 Not Found - PUT https://registry.npmjs.org/@mesh%2flang - Not found
 code: "E404"

      at /private/tmp/bunx-501-@morlay/bunpublish@latest/node_modules/npm-registry-fetch/lib/check-response.js:95:15
      at processTicksAndRejections (native:1:1)

Bun v1.1.27 (macOS x64)
error: script "release" exited with code 1
```

**Comment by morlay** [comment]
> @morlay i got same issue.
> 
> ```shell
> @mesh/lang:publish: 0.0.0
> 90 |         // www-authenticate header.
> 91 |         throw new errors.HttpErrorAuthOTP(
> 92 |           method, res, parsed, opts.spec
> 93 |         )
> 94 |       } else {
> 95 |         throw new errors.HttpErrorGeneral(
>                    ^
> HttpErrorGeneral: 404 Not Found - PUT https://registry.npmjs.org/@mesh%2flang - Not found
>  code: "E404"
> 
>       at /private/tmp/bunx-501-@morlay/bunpublish@latest/node_modules/npm-registry-fetch/lib/check-response.js:95:15
>       at processTicksAndRejections (native:1:1)
> 
> Bun v1.1.27 (macOS x64)
> error: script "release" exited with code 1
> ```

see fix https://github.com/morlay/bunpublish/issues/4#issuecomment-2261265242

404 of npm registry may means user not found when token missing.

**Comment by pipinet** [comment]
> > @morlay i got same issue.
> > ```shell
> > @mesh/lang:publish: 0.0.0
> > 90 |         // www-authenticate header.
> > 91 |         throw new errors.HttpErrorAuthOTP(
> > 92 |           method, res, parsed, opts.spec
> > 93 |         )
> > 94 |       } else {
> > 95 |         throw new errors.HttpErrorGeneral(
> >                    ^
> > HttpErrorGeneral: 404 Not Found - PUT https://registry.npmjs.org/@mesh%2flang - Not found
> >  code: "E404"
> > 
> >       at /private/tmp/bunx-501-@morlay/bunpublish@latest/node_modules/npm-registry-fetch/lib/check-response.js:95:15
> >       at processTicksAndRejections (native:1:1)
> > 
> > Bun v1.1.27 (macOS x64)
> > error: script "release" exited with code 1
> > ```
> 
> see fix [#4 (comment)](https://github.com/morlay/bunpublish/issues/4#issuecomment-2261265242)
> 
> 404 of npm registry may means user not found when token missing.

@morlay I have already added the configuration， still 404.

**Comment by mathix420** [comment]
> I managed to solve this. I updated my `bunfig.toml` to not use
> 
> ```toml
> [install.scopes]
> scope = { ... }
> ```
> 
> instead use
> 
> ```toml
> [install]
> registry = { url = "https://registry.npmjs.org", token = "$NPM_TOKEN" }
> ```
> 
> This works now perfectly fine for me 👍

Did fixed my issue, thanks again!
